test/driver_test.rb

```diff
@@ -78,12 +78,12 @@
     end
   end
 
-  describe "average_rating method" do
+  describe 'average_rating method' do
```

Comment:
`average_rating` and `total_revenue` should explicitly test the edge case of a driver that has trips in progress. You've done a great job addressing this in your methods.. make sure to test the functionality.